test/cypress/tests/dragnDrop.cy.ts

```diff
@@ -0,0 +1,368 @@
+import Header from '@editorjs/header';
+import Image from '@editorjs/simple-image';
+import * as _ from '../../../src/components/utils';
+
+describe('Drag and drop the block of Editor', function () {
+  beforeEach(function () {
+    cy.createEditor({
+      tools: {
+        header: Header,
+        image: Image,
+      },
+    }).as('editorInstance');
+
+    const numberOfBlocks = 3;
+
+    for (let i = 0; i < numberOfBlocks; i++) {
```

Comment:
Why don't you render the editor just with these blocks?

Reply:
nice idea